maint/docnotes

```diff
@@ -157,6 +157,12 @@ Deprecated routines (see 2.6.1)
  of those that was deprecated.  The routine may continue to be used, but new
  code should use the replacement routine.
 N*/
+Removed routines (see 2.6.1)
+/*N Removed
+ Removed Function:
+ The current MPI standard removed a number of routines.  This routine is one of
+ those that was removed.  The user should use the replacement routine.
```

Comment:
How about "The current MPI standard defines this routine as removed. The user should use the replacement routine."